### DISCUSSION TITLE
单卡单u实现单台日封装7T

### DISCUSSION THREAD

**Discussion by Chenhongwe** [filecoin-project/community-china#151] [矿机配置]
#### 机器推荐配置详情

- ###### Daemon

| 产品        | 数量/型号                        |
| :---------- | -------------------------------- |
| CPU         | 1 X AMD EPYC 7402P               |
| 内存        | 8 X 64GB/DDR4                    |
| 固态/数据盘 | 1 X SSD 8.0TB U.2                |
| GPU         | 2 X GeForce RTX 3080 涡轮        |
| sata        | 1 X SSD/512GB/M.2                |
| 主板        | 技嘉MZ32                         |
| 网卡        | 万兆双口网卡+模块                |
| 电源        | 4U8 12G扩展+1300W以上(冗余电源） |

- ###### Miner

| 产品        | 数量/型号                        |
| :---------- | -------------------------------- |
| CPU         | 1 X AMD EPYC 7402P               |
| 内存        | 8 X 64GB/DDR4                    |
| 固态/数据盘 | 1 X SSD 8.0TB U.2                |
| GPU         | 2 X GeForce RTX 3080 涡轮        |
| sata        | 1 X SSD/512GB/M.2                |
| 主板        | 技嘉MZ32                         |
| 网卡        | 万兆双口网卡+模块                |
| 电源        | 4U8 12G扩展+1300W以上(冗余电源） |

> **Daemon跟Miner共用一台，两台可互为主备，资金充裕的可各一台**

- ###### Worker（P1-C1）

| 产品        | 数量/型号                 |
| :---------- | ------------------------- |
| CPU         |1 X AMD EPYC 7542         |
| 内存        | 32X 64GB/DDR4             |
| 固态/数据盘 | 4 X SSD 8.0TB U.2         |
| GPU         | 1 X GeForce RTX 3080 涡轮 |
| sata        | 1 X SSD/512GB/M.2         |

- ###### Worker（C2）

| 产品        | 数量/型号                 |
| :---------- | ------------------------- |
| CPU         | 2 X AMD EPYC 7542         |
| 内存        | 16X 64GB/DDR4             |
| 固态/数据盘 | 1 X SSD/512GB/M.2         |
| GPU         | 6 X GeForce RTX 3080 涡轮 |
| sata        | 1 X SSD/512GB/M.2         |

- 以上配置结合我们优化后的软件单台C2封装效率为20T/台/天，P1-C1为7T/台/天，机器配比约为3:1

欢迎讨论交流，V：15088695255。

**Comment by zhaobo443** [comment]
FIL联合封装，用户提供质押币，公司提供设备和运维，收益60％,公司扣出20％服务费和20%设备费，合同期1080天，540天质押币退还，t3天封装。可实地考察。V:zhaobo9308
